src/styles/Modal.css

```diff
@@ -37,3 +37,41 @@
 .cancelDelete:hover {
   background-color: #ccc;
 }
+
+.detailsModal {
+  margin-bottom: 20px;
+  padding: 30px 20px;
+  background-color: #98c1d9;
+  border: 1px solid #293241;
+  border-radius: 6px;
+}
```

Comment:
Let's turn all the px into rem for responsiveness. Only use px if necessary (like border). Assume 10px = 1rem then adjust from there so it still looks similar.